src/pages/generate.tsx

```diff
@@ -129,6 +129,7 @@ export default function ClientIdentifierGenerator() {
     // Generate document from current form state and validate the field.
     const clientIdDocument = generateClientIdDocument({
       ...formValues,
+      contacts: formValues.contacts ? [formValues.contacts] : [],
```

Comment:
The `FormParameters` type should be updated so that `contacts` is a string array, and not a string. The ternary would become something like `formValues.contacts ? formValues.contacts : [],`

Reply:
https://github.com/inrupt/client-identifier-helper/pull/123